db/seeds.rb

```diff
@@ -1,3 +1,3 @@
-User.create!(email: "user@example.com", password: "12345678")
+User.create!(email: "user@example.com", password: "secret_password")
```

Comment:
Can we change this back to 12345678? :D We kinda use it for all our development apps. Muscle memory :tada: